POI/categories.go

```diff
@@ -51,6 +56,25 @@ func GetPlaceTypes(placeCat PlaceCategory) (placeTypes []LocationType) {
 	return
 }
 
+// Returns whether the eatory place is pricy based on its price level
+func IsPricyEatery(placeCategory PlaceCategory, priceLevel PriceLevel) bool {
+	if (placeCategory == PlaceCategoryEatery) && (priceLevel >= PriceLevelThree) {
+		return true
+	}
+	return false
+}
+
+// Generete Redis Key for Redis nearby search with place category and price info
+// The key includes the price level info for eatery and no price info for visit
+func EncodeNearbySearchRedisKey(placeCategory PlaceCategory, level PriceLevel) string {
```

Comment:
Nit: use lower case function name for internal functions.

Reply:
This function is actually called outside the package, so I keep the capitalized first letter of the function name.